[IMP] calendar: update authorization steps (#12163)
odoo/documentation: content/applications/productivity/calendar/google.rst

.. image:: google/new-api-project.png
   :alt: Create a new API project to store credentials.

> Give the API project a clear name, like 'Odoo Sync', so it can be identified. Then click the

Reviewer: Nitpick, please use literal formatting (backticks) for user input :)
```suggestion
Give the API project a clear name, like `Odoo Sync`, so it can be identified. Then click the
```

> Give the API Project a clear name, like "Odoo Sync", so it can be easily identified. Then click the

Reviewer: Lowercase "project", user input in literal formatting, and remove unnecessary adverbs please :) 
```suggestion
Give the API project a clear name, like `Odoo Sync`, so it can be identified. Then click the
```

   Note, as well, that while the API connection is in the *External* testing mode, then no approval
   is necessary from Google. User limits in this testing mode is set to 100 users.

> In the first step, :guilabel:`App Information`, type `Odoo` in the :guilabel:`App name` field, then
> enter the email address for the :guilabel:`User support email` field and click the :guilabel:`Next`
> button. In the second step, :guilabel:`Audience`, select :guilabel:`External`, then click the
> :guilabel:`Next` button. In the third step, :guilabel:`Contact Information`, enter your email again,
> then click the :guilabel:`Next` button. In the fourth step, :guilabel:`Finish`, tick the checkbox
> to agree to :guilabel:`Google API Services: User Policy.` For the last step, click the
> :guilabel:`Create` button.

Reviewer 2: Optional:
I think a numbered list would improve legibility here, instead of the paragraph (screenshot below)
![Screenshot 2025-02-24 at 2 03 00 PM](https://github.com/user-attachments/assets/df6203e5-793e-45f7-ad76-61277de926f7)

```suggestion
Follow the proceeding steps, in order of the wizard:

#. :guilabel:`App information`: type `Odoo` in the :guilabel:`App name` field, enter the email
   address for the :guilabel:`User support email` field, and then click the :guilabel:`Next` button.
#. :guilabel:`Audience`: select :guilabel:`External` and click the :guilabel:`Next` button.
#. :guilabel:`Contact Information`: enter the support email address again (used in step 1), and
   click the :guilabel:`Next` button
#. :guilabel:`Finish`: tick the checkbox to agree to :guilabel:`Google API Services: User Policy`
   and click the :guilabel:`Create` button.
```

Authorized domain Setup
-----------------------

Next, any domains set to appear on the consent screen or in an OAuth client's configuration must be
pre-registered. To do so, navigate to :guilabel:`Branding` in the left menu. In the
:guilabel:`Authorized domains` section, click the :guilabel:`Add domain` button to create a field to
enter an authorized domain. Enter a domain, such as `odoo.com`, then click the :guilabel:`Save`
button at the bottom of the page.

Test users
----------

> To give users the ability to sync with personal accounts, they must be set as a test user. Test

Reviewer 2: Consider defining "personal accounts" here— it's not clear what this means....i.e., are these email addresses that are not synced with the domain on the Odoo DB? Why would we give users the ability to do this, for what purpose(s)?

The writing here is also passive voice, can tighten it up so it's more active?

> users can be set up by clicking the :guilabel:`Add users` button in the :guilabel:`Test users`

Reviewer 3: ```suggestion
users can be set up by going to :guilabel:`Audience` in the left side menu and clicking the :guilabel:`Add users` button in the :guilabel:`Test users`
```

section. Enter any desired user emails, such as your own, and click the :guilabel:`Save` button.

Create credentials

> ------------------

Reviewer 3: On line 92, it's no longer "Credentials" in the left menu, but "Clients"

![image](https://github.com/user-attachments/assets/f39cf383-8cf1-43ef-b6bb-50b110a207da)

https://youtu.be/2incnX7VZk8?si=ADE3vGB-cZYxsg6N&t=280

   Note, as well, that while the API connection is in the *External* testing mode, then no approval
   is necessary from Google. User limits in this testing mode is set to 100 users.

> Follow the proceeding steps, in order:
> #. In :guilabel:`App Information`, type `Odoo` in the :guilabel:`App name` field, then enter the
> email address for the :guilabel:`User support email` field and click the :guilabel:`Next` button.
> #. In :guilabel:`Audience`, select :guilabel:`External`, then click the :guilabel:`Next` button.
> #. In :guilabel:`Contact Information`, enter your email again, then click the :guilabel:`Next`
> button.
> #. In :guilabel:`Finish`, tick the checkbox to agree to :guilabel:`Google API Services: User
> Policy.` For the last step, click the :guilabel:`Create` button.

Reviewer: Fix broken formatting please. Remove "your" pronoun
![Screenshot 2025-03-06 at 2 52 35 PM](https://github.com/user-attachments/assets/2145a27e-0753-43af-8b9c-75989a8a6393)

```suggestion
Follow the proceeding steps, in order:

#. In :guilabel:`App Information`, type `Odoo` in the :guilabel:`App name` field, then enter the
   email address for the :guilabel:`User support email` field and click the :guilabel:`Next` button.
#. In :guilabel:`Audience`, select :guilabel:`External`, then click the :guilabel:`Next` button.
#. In :guilabel:`Contact Information`, enter the email again, then click the :guilabel:`Next`
   button.
#. In :guilabel:`Finish`, tick the checkbox to agree to :guilabel:`Google API Services: User
   Policy.` For the last step, click the :guilabel:`Create` button.
```

> Authorized domain Setup

Reviewer: Please use sentence case headings
```suggestion
Authorized domain setup
```

-----------------------

Next, any domains set to appear on the consent screen or in an OAuth client's configuration must be
pre-registered. To do so, navigate to :guilabel:`Branding` in the left menu. In the
:guilabel:`Authorized domains` section, click the :guilabel:`Add domain` button to create a field to
enter an authorized domain. Enter a domain, such as `odoo.com`, then click the :guilabel:`Save`
button at the bottom of the page.

Test users
----------

To give users the ability to sync with personal Gmail accounts, they must be set as a test user.
Setup test users by going to :guilabel:`Audience` in the left-side menu and clicking the
:guilabel:`Add users` button in the :guilabel:`Test users` section. Enter any desired user emails,

> such as your own, and click the :guilabel:`Save` button.

Reviewer: Remove unnecessary pronoun use
```suggestion
and click the :guilabel:`Save` button.
```
